pkg/statuschecker/types/built_in.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/atlassian/smith/pkg/util"
 	sc_v1b1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
 	"github.com/pkg/errors"
+	apps "k8s.io/api/apps/v1"
```

Comment:
`apps_v1` to be consistent with other imports.

Reply:
Done